### ISSUE TITLE
Propose to remove (or redefine) Statement

### ISSUE THREAD

**Issue by adammontville** [sacmwg/draft-ietf-sacm-terminology#72]
We have defined *Statement* as "a subject defined in the SACM information model."

This feels too generic a term to be so specifically defined. In other words, the definition read alone would lead one to the term "SACM Subject" or something like that. Additionally, this term is also set up to be synonymous with *Subject*, which is defined elsewhere in this draft.

**Comment by henkbirkholz** [comment]
Actually the definition of Statement includes two paragraphs, I think. Te other one is.

"When a statement is used to provide content to a SACM domain, it is a top-level subject that bundles Content Elements into one subject and includes  metadata about the data origin."

If you want to merge both paragraphs into one, every improvement of the definition is welcome, if it retains the illustrated semantics.

**Comment by adammontville** [comment]
Yes, I've been striving to have one-sentence definitions of a particular form followed by expositional text, if necessary, as a second paragraph. The paragraph provides additional information about a statement, but does not add to the definition of statement (IMHO). 

I also cannot, for some reason, understand the "statement is a subject" part of the definition. And, given that the definition talks about the SACM information model specifically, should we really label this definition as "SACM Statement"? Still, I think the statement, if it concerns a subject defined in the SACM information model, ought to say something about that subject as it relates to a target endpoint (or something)...

Is it possible to use the following form? `A SACM Statement is a <fill in the blank here> that <fill in the refinement here>.`

**Comment by henkbirkholz** [comment]
I do not think that this is possible. Please have a look at RFC4949. Pressing everything into one sentence is sometimes possible for fundamental or axiomatic definitions. If you have a look at RFC 4949 you can see how difficult it is to define even the "most simple things", because that are in fact quite complex and their perceived simplicity often is more like a common misconception.

I.e. we tried not to excess on detail as RFC4949 does sometimes, but limit the scope on system entity characteristics vital to SACM and to new terms defined by SACM. One-sentence definitions in the scope of security automation are mostly unrealistic and should not be our target.

**Comment by henkbirkholz** [comment]
Section 4.3 of the SCAM IM version -10 illustrates the composition of the subject "sacm-statement":

[https://tools.ietf.org/html/draft-ietf-sacm-information-model-10#section-4.3](url)

**Comment by henkbirkholz** [comment]
I think the problem is that there are both Statement and SACM statement defined in the document. I am not sure if that was the initial reason to open this issue and maybe the source of my confusion.

My proposal is to remove the existing definition of SACM Statement and relabel Statement to SACM Statement.

Would this address this issue appropriately?
